blocks/related-blogs/related-blogs.js

```diff
@@ -76,32 +72,34 @@ async function getRelatedArticles(tag) {
 
 export default async function decorate(block) {
   // get the tag to be fetched
-  const tagConf = getConfiguredTag(readBlockConfig(block));
+  const { tag } = readBlockConfig(block);
 
   block.innerHTML = ''; // reset
 
-  const outerDiv = document.createElement('div');
-  outerDiv.classList.add('related-container');
-
-  // headline
-  const head4 = document.createElement('h4');
-  const text4head = document.createTextNode(`More ${tagConf}`);
-  head4.appendChild(text4head);
-  outerDiv.appendChild(head4);
-
-  outerDiv.appendChild(await getRelatedArticles(tagConf));
-  block.append(outerDiv);
-
-  const ovr = document.createElement('div');
-  ovr.classList.add('btn--light-teal');
-  ovr.classList.add('btn--solid');
-  ovr.classList.add('related-button-row');
-  const ovrlink = document.createElement('a');
-  ovrlink.href = '/insights';
-  ovrlink.classList.add('btn');
-  ovrlink.innerHTML = `BLOG OVERVIEW &nbsp; <i class="icons icon-wrapper ">
-    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 465 1024"><path d="M465.455 525.838L76.738 1020.226.001 966.133l319.528-455.391L.001 54.093 76.738 0l388.717 491.872z"></path></svg>
-    </i>`;
-  ovr.appendChild(ovrlink);
-  block.append(ovr);
+  const container = document.createElement('div');
+  container.classList.add('related-container');
+
+  // dynamic headline if filtered by tag
+  if (tag) {
+    const heading = document.createElement('h4');
+    heading.textContent = `More ${tag}`;
```

Comment:
Please put some todos with ticket reference in the code, hardcoding english translations will cause us a bit of trouble when migrating German.

Reply:
+1 we should you placeholders here https://www.hlx.live/docs/placeholders will, created #81 to track.